bobtemplates/plone/base.py

```diff
@@ -226,6 +229,8 @@ def set_global_vars(configurator):
         version = bob_config.version
     configurator.variables["plone.version"] = version
     set_plone_version_variables(configurator)
+    if hasattr(bob_config, "package_dottedname"):
```

Comment:
This needs to check that bob_config.package_dottedname is not empty, because if you set it for instance in `.mrbob.ini` and not in `bobtemplate.cfg` for instance, bob_config.package_dottedname exists but it is None so None is set in the variable